### PR TITLE
Included GC references for SPLASH garbage collection submission

### PR DESCRIPTION
**Pull request by jacob-hughes**

### CODE REVIEW

File: softdev.bib [hunk 1]
```diff
@@ -2060,3 +2060,39 @@ @article{boehm1988garbage
     publisher={Wiley Online Library},
     month=Sep
 }
+
+@article{boehm1993space,
+    author = {Boehm, Hans-Juergen},
+    title = {Space Efficient Conservative Garbage Collection},
+    journal = {SIGPLAN Not.},
```
Comment (ltratt) [inline softdev.bib]: This was a conference paper at PLDI.

File: softdev.bib [hunk 2]
```diff
@@ -2060,3 +2060,39 @@ @article{boehm1988garbage
...
+    month = Jun,
+    pages = {197--206},
+    year = {1993},
+    publisher = {ACM},
+}
+
+@article{blackburn2003ulterior,
+    author = {Blackburn, Stephen M. and McKinley, Kathryn S.},
+    title = {Ulterior Reference Counting: Fast Garbage Collection Without a Long Wait},
+    journal = {SIGPLAN Not.},
```
Comment (ltratt) [inline softdev.bib]: This was at OOPSLA.

File: softdev.bib [hunk 3]
```diff
@@ -2060,3 +2060,39 @@ @article{boehm1988garbage
...
+    month = Oct,
+    year = {2003},
+    pages = {344--358},
+    publisher = {ACM},
+}
+
+@article{blackburn2004myths,
+    author = {Blackburn, Stephen M. and Cheng, Perry and McKinley, Kathryn S.},
+    title = {Myths and Realities: The Performance Impact of Garbage Collection},
+    journal = {SIGMETRICS Perform. Eval. Rev.},
```
Comment (ltratt) [inline softdev.bib]: I'm not sure what conference this was in -- "International Conference on Measurement and Modeling of Computer Systems"?

**Comment by ltratt** [comment]
I think all 3 of these should be `inproceedings' and not have the `journal` tag?

**Comment by jacob-hughes** [comment]
This should pass the prebib test now and be ready for squashing.

File: softdev.bib [hunk 4]
```diff
@@ -2060,3 +2060,39 @@ @article{boehm1988garbage
...
+    month = Oct,
+    year = {2003},
+    pages = {344--358},
+    publisher = {ACM},
+}
+
+@inproceedings{blackburn2004myths,
+    author = {Blackburn, Stephen M. and Cheng, Perry and McKinley, Kathryn S.},
+    title = {Myths and Realities: The Performance Impact of Garbage Collection},
+    booktitle = {International Conference on Measurement and Modeling of Computer Systems},
```
Comment (ltratt) [inline softdev.bib]: It turns out this is abbreviated to `ICMMCS` https://waset.org/conference/2018/09/vancouver/ICMMCS.

**Comment by jacob-hughes** [comment]
Ready for squashing (I think)

**Comment by ltratt** [comment]
Please squash.

**Comment by jacob-hughes** [comment]
Squashed

**Comment by ltratt** [comment]
One minor comment: in the commit message, we don't need to say what the bib entries are used *for* (i.e. just because they're for your current purpose isn't very important: other people will probably use them for other purposes in the future). Feel free to reword and force push.

**Comment by jacob-hughes** [comment]
Okay, I've reworded this now

**Comment by ltratt** [comment]
Thanks!

**Pull request merged**